### PR TITLE
PAB-2872: Copy over important C8Y-specific info from PAM release notes

### PR DESCRIPTION
**Pull request by kailol-sag**
This has been copied over from the following topics in the published Apama 10.11.1 release notes:
https://documentation.softwareag.com/apama/v10-11/apama10-11/apama-webhelp/index.html#page/apama-webhelp%2Fco-ApaRelNot_10111_misc_backwards_compatibility.html
https://documentation.softwareag.com/apama/v10-11/apama10-11/apama-webhelp/index.html#page/apama-webhelp%2Fco-ApaRelNot_10111_miscellaneous.html

### CODE REVIEW

File: content/release-10-13-0/announcements-10-13-0.md [hunk 1]
```diff
@@ -132,3 +132,21 @@ The API to download the serialized source of a PMML model `{{url}}/service/zemen
...
+##### Cumulocity IoT transport in Apama
+
+The following applies as of Apama 10.11.1:
+
+- The `GeoFenceContainer.createGeoFenceContainer()` action no longer throws an exception when provided with any `lat` or `lng` values that cannot be parsed as a float. 
+  Instead, an empty `optional<GeoFencePoint>` is now created for that coordinate. 
+  See the `com.apama.cumulocity` package in the [API Reference for EPL (ApamaDoc)](https://documentation.softwareag.com/apama/v10-11/apama10-11/ApamaDoc/index.html) 
+  for more information on the `GeoFenceContainer` event.
+
+- Warning messages are now logged for Cumulocity IoT queries where `pageSize` is below 50 and `currentPage` is not set (default).
```
Comment (skom-sag) [inline content/release-10-13-0/announcements-10-13-0.md:149]: should we say 'Cumulocity IoT' or use a label here?

File: content/release-10-13-0/announcements-10-13-0.md [hunk 2]
```diff
@@ -132,3 +132,21 @@ The API to download the serialized source of a PMML model `{{url}}/service/zemen
...
+
+- The `GeoFenceContainer.createGeoFenceContainer()` action no longer throws an exception when provided with any `lat` or `lng` values that cannot be parsed as a float. 
+  Instead, an empty `optional<GeoFencePoint>` is now created for that coordinate. 
+  See the `com.apama.cumulocity` package in the [API Reference for EPL (ApamaDoc)](https://documentation.softwareag.com/apama/v10-11/apama10-11/ApamaDoc/index.html) 
+  for more information on the `GeoFenceContainer` event.
+
+- Warning messages are now logged for Cumulocity IoT queries where `pageSize` is below 50 and `currentPage` is not set (default). 
+  This is because setting a small `pageSize` without setting `currentPage` can result in queries that run very slowly. 
+  For example, to request the first 20 items, you have to set `pageSize` to 20 and `currentPage` to 1. 
+  See also [Paging Cumulocity IoT queries](https://documentation.softwareag.com/apama/v10-11/apama10-11/apama-webhelp/index.html#page/apama-webhelp%2Fco-ConApaAppToExtCom_cumulocity_paging_cumulocity_queries.html) in the Apama documentation.
```
Comment (skom-sag) [inline content/release-10-13-0/announcements-10-13-0.md:152]: here too

File: content/release-10-13-0/announcements-10-13-0.md (hunk 1 above)
Comment (BeateRixen) [reply content/release-10-13-0/announcements-10-13-0.md:149]: We usually don´t use quotes are other highlighting for the term Cumulocity IoT.

**Review by BeateRixen — APPROVED** [review]

File: content/release-10-13-0/announcements-10-13-0.md (hunk 1 above)
Comment (skom-sag) [reply content/release-10-13-0/announcements-10-13-0.md:149]: I am not referring to adding quotes. I am thinking of usecase where customers like DT would like to replace cumulocity iot with a different name.

We have substitution strings for such terms that are maintained centrally right, so should we be using them?

Comment (BeateRixen) [reply content/release-10-13-0/announcements-10-13-0.md:149]: Yes, we use short codes in other places but not in the release notes so far. DT told me that they create their own release notes and not use our sources.

Comment (skom-sag) [reply content/release-10-13-0/announcements-10-13-0.md:149]: ok, great

**Review by skom-sag — APPROVED** [review]

**Pull request merged**